src/reference/90-Developers-Guide/09-Launcher/01-Launcher-Getting-Started.md

```diff
@@ -154,7 +163,9 @@ class Main extends xsbti.AppMain
 Next, define a configuration file for the launcher. For the above class,
 it might look like:
 
-Then, `publishLocal` or `+publishLocal` the application to make it
+FIXME
```

Comment:
Okay ditto, this was also in the 0.12 docs and lost somehow:

```
[scala]
  version: 2.9.2
[app]
  org: org.scala-sbt
  name: xsbt-test
  version: 0.12.0
  class: xsbt.test.Main
  cross-versioned: true
[repositories]
  local
  maven-central
[boot]
 directory: ${user.home}/.myapp/boot
```

The preceding code example needs updating to depict a current Scala version there. I assume that `0.12.0` is an arbitrary version number for the application itself, not actually an sbt version as it sort of suggests?

I'll add this back on the branch.